readme.md

```diff
@@ -320,6 +320,8 @@ Some of our docs have tests that run only in CI by default. If you end up needin
 ## Lighthouse Integrations in Web Perf services
 This section details services that have integrated Lighthouse data. If you're working on a cool project integrating Lighthouse and would like to be featured here, file an issue to this repo or tweet at us [@_____lighthouse](https://twitter.com/____lighthouse)!
 
+* **[Alertdesk](https://www.alertdesk.com/)** - Alertdesk is based on Lighthouse and helps you to keep track of your site’s quality & performance. Run daily quality & performance tests from both Mobile and Desktop and dive into the powerful & intuitive reports. Alertdesk also monitors your uptime (every minute - 24/7) & domain health.  
```

Comment:
mention paid product.

Reply:
Thank you for the quick response. I've added the "paid product"-part to the description.